src/components/OptionList/components/Checkbox/Checkbox.scss

```diff
@@ -3,64 +3,145 @@
 .Checkbox {
   position: relative;
   width: 100%;
+  margin: var(--p-choice-margin, 0);
 }
 
 .Input {
   @include visually-hidden;
+}
+// stylelint-disable selector-max-specificity, selector-max-class, selector-max-combinators, max-nesting-depth
+.Checkbox:not(.globalTheming) {
```

Comment:
It's unfortunate but the Checkbox is pretty much duplicated because AutoComplete. Checkboxes should be used for comboboxes, in the new version of Autocomplete this will be able to change.